lepton/image.go

```diff
@@ -95,11 +95,11 @@ func addPasswd(m *fs.Manifest, c *types.Config) {
 	data := []byte("root:x:0:0:root:/root:/bin/nobash")
 	err := os.WriteFile(passwd, data, 0644)
 	if err != nil {
-		panic(err)
+		fmt.Printf("Failed save passwd in temporary file, error: %s", err)
 	}
 	err = m.AddFile("/etc/passwd", passwd)
 	if err != nil {
-		panic(err)
+		fmt.Printf("Failed add passwd, error: %s", err)
 	}
```

Comment:
since this was a panic need to exit here

Reply:
done